website/i18n/ko/docusaurus-plugin-content-docs/current/widgets/table_cell.md

```
@@ -0,0 +1,30 @@
+# TableCell
+StacTableCell을 사용하면 JSON을 사용하여 Flutter TableCell 위젯을 정의할 수 있습니다.
+Flutter의 TableCell 위젯에 대해 자세히 알아보려면 [공식 문서](https://api.flutter.dev/flutter/widgets/TableCell-class.html)를 참조하세요.
+
+## Properties 
+| Property          | Type                          | Description                                                                                                                                   |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)